JsonClient/JsonClient/Extensions/PropertyInfoExtensions.cs

```diff
@@ -13,6 +14,12 @@ public static class PropertyInfoExtensions
             }
             else            
                 return pi.Name;
-        }
+    }
+    public static string GetDisplayName(this Enum value)
```

Comment:
This method should be part of a different class because doesn't know anything about PropertyInfo